unit-tests/src/test/java/org/eclipse/collections/impl/list/IntervalTest.java

```diff
@@ -57,6 +58,11 @@ public void fromAndToAndBy()
         Verify.assertEqualsAndHashCode(interval, Interval.fromTo(1, 1));
         Verify.assertEqualsAndHashCode(interval2, Interval.fromTo(1, 10));
         Verify.assertEqualsAndHashCode(interval3, Interval.fromToBy(1, 10, 2));
+
+        Verify.assertThrows(IllegalArgumentException.class, () -> Interval.fromTo(Integer.MIN_VALUE, Integer.MAX_VALUE));
+        Verify.assertThrows(IllegalArgumentException.class, () -> Interval.fromTo(-1, Integer.MAX_VALUE));
```

Comment:
there should be more test cases:
1) Interval.fromTo(Integer.MIN_VALUE, 1)
2) Interval.fromTo(1, Integer.MIN_VALUE, -1)
3) Interval.fromTo(Integer.MAX_VALUE, -1, -1)

These should not throw:
1) Interval.fromTo(Integer.MIN_VALUE, 0)
2) Interval.fromTo(0, Integer.MAX_VALUE)

Reply:
These tests are still missing or I missed reading them?